test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -257,114 +242,82 @@ def api_client
       creds = GRPC::Core::CallCredentials.new(authentication.updater_proc)
       ssl_creds.compose(creds)
 
-      # Here we have obtained the creds, but for the mock, we will leave the
-      # channel insecure.
-      @grpc_stub.new(@mock_host, :this_channel_is_insecure)
+      @grpc_stub
     end
   end
 
   # GRPC logging mock that successfully logs the records.
-  class GRPCLoggingMockService < Google::Logging::V2::LoggingServiceV2::Service
+  class GRPCLoggingMockService <
+      Google::Cloud::Logging::V2::LoggingServiceV2Client
     def initialize(requests_received)
       super()
       @requests_received = requests_received
     end
 
-    def write_log_entries(request, _call)
+    def write_log_entries(entries,
+                          log_name: nil,
+                          resource: nil,
+                          labels: nil)
+      request = Google::Apis::LoggingV2::WriteLogEntriesRequest.new(
+        log_name: log_name,
+        resource: resource,
+        labels: labels,
+        entries: entries
+      )
       @requests_received << request
       WriteLogEntriesResponse.new
     end
-
-    # TODO(lingshi) Remove these dummy methods when grpc/9033 is fixed.
-    #
-    # These methods should never be called, so they will just fail the tests
-    # with "unimplemented" errors..
-    def _undefined
-      raise "Method #{__callee__} is unimplemented and needs to be overridden."
-    end
-
-    alias list_logs _undefined
-    alias list_log_entries _undefined
-    alias list_log_services _undefined
-    alias list_log_service_indexes _undefined
-    alias list_monitored_resource_descriptors _undefined
-    alias delete_log _undefined
-    undef_method :_undefined
   end
 
   # GRPC logging mock that fails and returns server side or client side errors.
   class GRPCLoggingMockFailingService <
-      Google::Logging::V2::LoggingServiceV2::Service
-    # 'code_sent' and 'message_sent' are references of external variables. We
-    #  will assert the values of them later. 'code_value' and 'message_value'
-    #  are actual error code and message we expect this mock to return.
+      Google::Cloud::Logging::V2::LoggingServiceV2Client
     def initialize(code, message, failed_attempts)
       @code = code
       @message = message
       @failed_attempts = failed_attempts
       super()
     end
 
-    def write_log_entries(_request, _call)
+    # rubocop:disable Lint/UnusedMethodArgument
+    def write_log_entries(entries,
```

Comment:
This whole signature should fit on one line.

Reply:
Changed.